doc/ArchInstall/arch_install.sh

```diff
@@ -29,9 +29,11 @@ mount ${ROOT} /mnt
 mkdir -p /mnt/boot/efi
 mount ${BOOT} /mnt/boot/efi
 
-# Bootstrap Pacman, and trust everything to prevent pgp key errors
+# Bootstrap Pacman
 pacstrap /mnt base linux linux-firmware
-cp /etc/pacman.conf /mnt/etc/pacman.conf
+# Set mirrorlist to match Dockerfile
+# ↓↓↓ UPDATE MIRRORS HERE ↓↓↓
+echo -e "Server=https://archive.archlinux.org/repos/2022/07/10/\$repo/os/\$arch" | tee /etc/pacman.d/mirrorlist
```

Comment:
I think this would be better if you made a file at `docker/etc/pacman.d/mirrorlist` that had the correct contents and then you can `wget` it when installing here and you can update the Dockerfile to copy that file into the docker image. That way there is only 1 location where the mirrorlist is set and you can't forget to update it somewhere

Reply:
Thanks, that sounds like a good way to do it. I've also added updating that file as an option to systemconfigurator. I'm not sure if it's better to do it that way or just copy the file, but I do think it should be in systemconfigurator somewhere.